aea/helpers/base.py

```diff
@@ -174,77 +172,6 @@ def load_aea_package(configuration: ComponentConfiguration) -> None:
         spec.loader.exec_module(module)  # type: ignore
 
 
-def load_all_modules(
```

Comment:
Interesting how much code we no longer need ...

Reply:
a lot of it was just not used anymore, and it has not been removed.

Reply:
moreover, `load_all_modules` was more generic, hence it needed other code to work.

Reply:
and, btw, we are still far from the ideal loading mechanism. We still don't handle correctly the conflicting versions for packages of different AEA projects in the same process...

Reply:
I think we decided we won't handle that at all. We have the requirement that there is one version per process; that's it. Totally ok as its also the case with any other python package.